index.html

```diff
@@ -116,17 +115,18 @@ <h2>Main Program<hr></h2>
                     </div> 
```

Comment:
- [ ] In this `<section class="main"> section, kindly watch out for these suggestions to improve the experience and aesthetics of the page:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/251971148-2540cf9a-8a09-4d68-adac-c6283f5887fc.png)|![image](https://user-images.githubusercontent.com/99715640/251971140-9e655fd0-2c02-469d-a789-6a6e44e42745.png)|

  - [ ] Kindly ensure that the entire section is well aligned to the center in both views
  - [ ] I think your `gap` value on the `display: flex` is too much than the design blueprint provided
  - [ ] I noticed the use of font-weight of 700 on the `p` elements. Kindly remove or make it 400 to lighten things as shown in Cindy's design
  - [ ] Again, it appears that you need to reduce the `line-height` as well as the `letter-spacing`
  - [ ] Lastly, please mind the spacing around all elements, not collapsing elements into another.

Reply:
Good